labs/lab-02/tasks/pointers/support/main.c

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <stdio.h>
+#include <string.h>
+#include <assert.h>
+#include <stdlib.h>
+
+#include "pointers.h"
+
+int main(void)
+{
+	char s1[] = "Abracadabra";
+	char s2[] = "Abracababra";
+	char src[] = "Learn IOCLA, you must!";
```

Comment:
```suggestion
	char src[] = "Learn HSI, you must!";
```
Another thing that we missed during translation. Make sure this is consistent throughout all files.